plugin/completion/base_complete.py

```diff
@@ -120,16 +120,20 @@ def init(self, view, settings):
 
         """
         if not view:
+            log.critical(" no view to initialize completer!")
             return
         current_dir = path.dirname(view.file_name())
         search_scope = SearchScope(
             from_folder=current_dir,
             to_folder=settings.project_base_folder)
         self.flags_manager = FlagsManager(
-            use_cmake=settings.generate_flags_with_cmake,
-            flags_update_strategy=settings.cmake_flags_priority,
-            cmake_prefix_paths=settings.cmake_prefix_paths,
+            view=view,
+            settings=settings,
+            compiler_variant=self.compiler_variant,
```

Comment:
It would be a regression if it would not be like that before. :) Just before the flags manager was using the same code to initialize both in `bin_completer` and in `lib_completer` but essentially, the behavior was similar.